docs/shared/react-tutorial/react-monorepo.md

```diff
@@ -0,0 +1,827 @@
+              },
+              {
+                "sourceTag": "type:feature",
+                "onlyDependOnLibsWithTags": ["type:feature", "type:ui"]
+              },
+              {
+                "sourceTag": "type:ui",
+                "onlyDependOnLibsWithTags": ["type:ui"]
+              },
+              {
+                "sourceTag": "scope:orders",
+                "onlyDependOnLibsWithTags": [
+                  "scope:orders",
+                  "scope:products",
+                  "scope:shared"
+                ]
+              },
+              {
+                "sourceTag": "scope:products",
+                "onlyDependOnLibsWithTags": ["scope:products", "scope:shared"]
+              },
+              {
+                "sourceTag": "scope:shared",
+                "onlyDependOnLibsWithTags": ["scope:shared"]
+              }
+            ]
+          }
+        ]
+      }
+    },
+    ...
+  ]
+}
+```
+
+To test it, go to your `libs/products/src/lib/product-list/product-list.tsx` file and import the `OrderList` from the `orders` project:
+
+```tsx {% fileName="libs/products/src/lib/product-list/product-list.tsx" %}
+import styles from './product-list.module.css';
+
+// This import is not allowed 👇
+import { OrderList } from '@react-monorepo/orders';
+
+/* eslint-disable-next-line */
+export interface ProductListProps {}
+
+export function ProductList(props: ProductListProps) {
+  return (
+    <div className={styles['container']}>
+      <h1>Welcome to ProductList!</h1>
+      <OrderList />
+    </div>
+  );
+}
+
+export default ProductList;
+```
+
+If you lint your workspace you'll get an error now:
+
+```{% command="nx run-many -t lint" %}
+    ✔  nx run myreactapp:lint  [existing outputs match the cache, left as is]
+    ✔  nx run e2e:lint  [existing outputs match the cache, left as is]
+    ✔  nx run shared-ui:lint (1s)
+
+    ✖  nx run products:lint
+       Linting "products"...
+
+       /Users/.../react-monorepo/libs/products/src/lib/product-list/product-list.tsx
+         3:1  error  A project tagged with "scope:products" can only depend on libs tagged with "scope:products", "scope:shared"  @nx/enforce-module-boundaries
+
+       ✖ 1 problem (1 error, 0 warnings)
+
+       Lint errors found in the listed files.
+
+    ✔  nx run orders:lint (1s)
+
+ ————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
+
+ >  NX   Ran target lint for 5 projects (1s)
+
+    ✔    4/5 succeeded [2 read from cache]
+
+    ✖    1/5 targets failed, including the following:
+         - nx run products:lint
+```
+
+If you have the ESLint plugin installed in your IDE you should immediately see an error:
+
+![ESLint module boundary error](/shared/images/tutorial-react-standalone/react-standalone-module-boundaries.png)
+
+Learn more about how to [enforce module boundaries](/core-features/enforce-module-boundaries).
+
+## Next Steps
```

Comment:
on the linear issue, I also had a section "Setting up CI". Any reason that was skipped here?